main/query.go

```diff
@@ -37,7 +37,7 @@ func main() {
 	}
 
 	apiInstance := common.NewAPI()
-	myBackend := blueflood.NewBlueflood(*common.BluefloodUrl, *common.BluefloodTenantId)
+	myBackend := blueflood.NewBlueflood(blueflood.BluefloodClientConfig{*common.BluefloodUrl, *common.BluefloodTenantId})
```

Comment:
It looks like `main/query.go` got skipped over somehow, so it's not currently in master.